test/app/Intro Notebook.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().system('pip install quandl')
+
+
+# In[3]:
+
+
+import quandl
+quandl.ApiConfig.api_key = '38VWdHpGVLZs94hZCMC1'
```

Comment:
Can you remove the API key from the code. Instead you should use an environment variable using:

```python
import os
quandl.ApiConfig.api_key =  os.environ["QUANDL_API_KEY"]
```